src/main/scala/parsley/Cont.scala

```diff
@@ -59,7 +59,7 @@ private [parsley] object Cont
     }
 }
 
-private [parsley] class Id[_, A](var x: A)
+private [parsley] class Id[R, A](var x: A)
```

Comment:
Dotty doesn't like underscores for phantom types.

Reply:
I think there's going to be an issue with `private [this]` at some point, but I'll defer that till it's a problem